src/impl/sctptransport.cpp

```diff
@@ -284,6 +284,16 @@ SctpTransport::SctpTransport(shared_ptr<Transport> lower, const Configuration &c
 		throw std::runtime_error("Could not disable SCTP fragmented interleave, errno=" +
 		                         std::to_string(errno));
 
+	// When using SCTP over DTLS, the data integrity is ensured by DTLS, so there's no need to
+	// additionally compute CRC32c.
+	// See https://datatracker.ietf.org/doc/html/draft-ietf-tsvwg-sctp-zero-checksum
+	if (config.sctpZeroChecksum) {
```

Comment:
The feature should be enabled by default as it handles retro-compatibility automatically.